src/iris/ui/static/js/iris.js

```diff
@@ -244,8 +246,35 @@ iris = {
       data.$page.on('change', data.appSelect, this.renderVariables.bind(this));
       data.$page.on('change', data.testPlanInputs, this.updateTestPlanValues);
       data.$page.on('change', data.toggleDynamic, this.toggleDynamicTargets);
+      data.$page.on('change', data.waitInput, this.updateStepTimes.bind(this))
+      data.$page.on('change', data.countInput, this.updateStepTimes.bind(this))
       window.onbeforeunload = iris.unloadDialog.bind(this);
     },
+    updateStepTimes: function() {
+      var self = this;
+      $('.step-time').each(function(_, element) {
+        self.updateStepTime(element)
+      });
+    },
+    updateStepTime: function(element) {
+      var $element = $(element),
+          waits = [],
+          stepTime;
+
+      $element.parent().siblings('.plan-notification').each(function(_, n) {
+        var $n = $(n);
+        if ($n.attr('data-mode') === 'view') {
+          waits.push($n.children('.notification-wait')[0].title * $n.children('.notification-count')[0].title);
+        } else {
+          waits.push($n.find('.notification-wait')[0].value * $n.find('.notification-count')[0].value);
+        }
+      })
+      stepTime = Math.min(...waits);
```

Comment:
I don't think this spread syntax is supported everywhere: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax

To keep consistent with the rest of the iris JS not using new es6+ features, can we make this use a non-spread equivalent?